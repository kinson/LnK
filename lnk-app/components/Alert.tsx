import React, { ReactElement } from 'react';

interface Props {
  children: string;
}

export default function Alert({ children }: Props): ReactElement {
  return (
    <p className="my-8 rounded-md bg-red-200 border-2 border-red-400 text-red-700 p-4">
      {children}
    </p>
  );
}
