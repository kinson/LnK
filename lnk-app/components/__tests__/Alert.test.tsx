import React from 'react';
import renderer from 'react-test-renderer';
import Alert from '../Alert';

describe('Alert component tests', () => {
  it('should render the Alert component with a message', () => {
    const component = renderer.create(
      <Alert>Could not find the URL you are looking for.</Alert>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
