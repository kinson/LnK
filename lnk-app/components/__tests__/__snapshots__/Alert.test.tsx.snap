// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert component tests should render the Alert component with a message 1`] = `
<p
  className="my-8 rounded-md bg-red-200 border-2 border-red-400 text-red-700 p-4"
>
  Could not find the URL you are looking for.
</p>
`;
